From 6999314a1ea3b31fff856ab5c8b4d7941a1b455d Mon Sep 17 00:00:00 2001
From: Varsha Maira <vmaira@habana.ai>
Date: Wed, 21 May 2025 14:21:37 +0300
Subject: [PATCH] [SW-218933] g3: specs: move enum mem_acces_mode to fs1 specs

Currently, the `mem_access_mode` enum is defined within individual Zephyr
apps.

This patch moves the enum to a common location under the `specs` directory
to enable reuse across multiple apps and avoid definitions in multiple
apps for fs1.

Change-Id: Ifa3f3b11d0be6ecefec2d17eafbb4fbdc383de57
Signed-off-by: Varsha Maira <vmaira@habana.ai>
---

diff --git a/fs/common/hl_sec_if.h b/fs/common/hl_sec_if.h
index 7777f24..a1481f7 100644
--- a/fs/common/hl_sec_if.h
+++ b/fs/common/hl_sec_if.h
@@ -76,4 +76,10 @@
 	uint32_t efuse_sys_cfg2;	/* Efuse security data at offset 0x3B */
 };
 
+enum mem_access_mode {
+	ACCESS_RO,
+	ACCESS_WR,
+	ACCESS_RW,
+};
+
 #endif /* _HABANALABS_SEC_IF_H_ */
